Makefile

```diff
@@ -240,7 +242,7 @@ endif
 # only tsh is built.
 #
 .PHONY:full
-full: $(ASSETS_BUILDDIR)/webassets
+full: $(WEB_ASSETS_BUILDDIR)
 ifneq ("$(OS)", "windows")
 	$(MAKE) all WEBASSETS_TAG="webassets_embed"
```

Comment:
With this change can we get away from the `webassets_embed` build tag, or is there still a good reason to keep the tag?  I am not a fan of the build tag because I prefer to `go install tool/teleport` and then it inevitably fails to run due to missing web assets.

Reply:
I'm not exactly sure what that entails, but I think the asset handling in general could be cleaned up from its current state. I would make an issue for your specific issue and handle it in another PR.